Atomic/syscontainers.py

```diff
@@ -963,29 +982,29 @@ def _do_checkout(self, repo, name, img, mode, deployment, values, destination, u
         else:
             systemd_template = SYSTEMD_UNIT_FILE_DEFAULT_TEMPLATE
 
-        values = self._amend_values(values, manifest, name, img, image_id, destination, prefix, unit_file_support_pidfile=SystemContainers._template_support_pidfile(systemd_template))
+        values = self._amend_values(options["values"], manifest, options["name"], options["img"], image_id, options["destination"], options["prefix"], unit_file_support_pidfile=SystemContainers._template_support_pidfile(systemd_template))
 
         src = os.path.join(exports, "config.json")
-        destination_path = os.path.join(destination, "config.json")
+        destination_path = os.path.join(options["destination"], "config.json")
         if os.path.exists(src):
             shutil.copyfile(src, destination_path)
         elif os.path.exists(src + ".template"):
             with open(src + ".template", 'r') as infile:
                 util.write_template(src + ".template", infile.read(), values, destination_path)
```

Comment:
And ofc... there is another duplicated values here, that makes the compiler not show warnings #facepalms